heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesController.java

```diff
@@ -14,26 +14,34 @@
 
 package com.twitter.heron.scheduler.kubernetes;
 
+import java.io.IOException;
 import java.net.HttpURLConnection;
-import java.util.Arrays;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import com.twitter.heron.spi.utils.NetworkUtils;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import com.twitter.heron.common.network.HttpJsonClient;
+import com.twitter.heron.scheduler.TopologySubmissionException;
 
 public class KubernetesController {
   private static final Logger LOG = Logger.getLogger(KubernetesController.class.getName());
 
   private final String kubernetesURI;
```

Comment:
Looks like kubernetesURI is only used in the constructor now so this can be removed.